src/Trixi.jl

```diff
@@ -49,6 +49,7 @@ using Octavian: Octavian, matmul!
 using Polyester: @batch # You know, the cheapest threads you can find...
 using OffsetArrays: OffsetArray, OffsetVector
 using P4est
+@reexport using NLsolve: nlsolve
```

Comment:
This package provides a Newton's method. This is required when working with the equivalent potential temperature. It is only used in the elixirs and hence could be moved to the respective elixir files.

Reply:
If only used in the elixirs, this should be moved there.